### ISSUE TITLE
Reversible airflow obstruction

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6549]
**Preferred term label:** Reversible airflow obstruction

**Synonyms**
Reversible pulmonary obstruction

**Definition (free text, please give PubMed ID)**
Airflow obstruction with a  significant response to a bronchodilator defined as an increase in  FEV1 and/or FVC by 12 percent or more and by at least 200 mL. PMID: 16264058

**Parent term (use hpo.jax.org/app)**
Pulmonary obstruction HP:0006536

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Can we discuss how we should model pulmomary obstruction vs airflow obstruction in the ontology?

**Comment by pnrobinson** [comment]
added
